lib/git-revision-view.coffee

```diff
@@ -129,6 +114,10 @@ class GitRevisionView
       editor1: newTextEditor    # the older revision
       editor2: editor           # current rev
 
+    SplitDiff.setConfig 'leftEditorColor', 'green'
+    SplitDiff.setConfig 'leftEditorColor', 'red'
```

Comment:
Ahhh...yes, I think so. I had it that way originally, but forgot to change it back. Do you want another PR for that since thus one is merged?